### PR TITLE
Customizable logo size, site_title, and background images

### PR DESCRIPTION
**Pull request by hao-fang**
This will make it easier to use conference-specific configs, e.g., https://github.com/acl-org/acl-2020-virtual-conference/pull/15.

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -91,8 +91,19 @@
     >
       <div class="container">
         <a class="navbar-brand" href="index.html">
-          <img class="logo" src="{{config.logo}}" width="180px" />
+          <img
+             class="logo" src="{{config.logo.image}}"
+             {% if config.logo.height %}
```
Comment (srush) [inline templates/base.html:96]: Probably better to just use https://jinja.palletsprojects.com/en/2.11.x/templates/#if-expression for these sort of expressions.

Comment (srush) [reply templates/base.html:96]: Oh actually this is even better https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#defaulting-undefined-variables

{{ config.image.width | default("") }}

Comment (hao-fang) [reply templates/base.html:96]: :wow:

File: templates/base.html [hunk 2]
```diff
@@ -91,8 +91,15 @@
     >
       <div class="container">
         <a class="navbar-brand" href="index.html">
-          <img class="logo" src="{{config.logo}}" width="180px" />
+          <img
+             class="logo" src="{{config.logo.image}}"
+             height={{config.logo.height | default("auto") | tojson}}
+             width={{config.logo.width | default("auto") | tojson}}
+          />
```
Comment (hao-fang) [inline templates/base.html:98]: use default magic and also added "tojson" to make sure the string is surrounded by double quotes.
The generated html file looks like
```html
        <a class="navbar-brand" href="index.html">
          <img
             class="logo" src="static/images/MiniConf.png"
             height="auto"
             width="180px"
          />
        </a>

```

**Pull request merged**